lib/index.js

```diff
@@ -9,6 +9,10 @@ module.exports = function (options) {
     options.classIgnore = []
   }
 
+  if (!options.extensionIgnore) {
```

Comment:
explicit comparison always works faster than type coercion and comparison.

better
```js
options.extensionIgnore === undefined
```

Reply:
Totally agreeing, I was following the the `classIgnore`

```javascript
  if (!options.classIgnore) {
    options.classIgnore = []
  }
```

Reply:
Done, in both actually 🤓